plugins/multilevel-linear-view/src/MultilevelLinearComparativeView/components/Controls.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Typography, Button, FormGroup, useTheme, alpha } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { SearchBox } from '@jbrowse/plugin-linear-genome-view'
+
+import { LinearGenomeMultilevelViewModel } from '../../LinearGenomeMultilevelView/model'
+import { MultilevelLinearComparativeViewModel } from '../model'
+
+type LCV = MultilevelLinearComparativeViewModel
```

Comment:
I was not aware of MultilevelLinearComparativeView, what does it look like?

Reply:
It's basically the MLLV -- it fits inside the MultilevelLinearView component with the import form (i.e. when the import form is completed the compartiveviewcomponent is displayed with the relevant properties)

the name is inherited from linear-comparative-view when i was constructing mllv, i can change it to something more appropriate if needed but i don't think it matters much